### PR TITLE
[EVIL CRAB ] pda-b-gone deployed

### PR DESCRIPTION
**Pull request by RatFromTheJungle**
## About The Pull Request
This re-removes the generic PDA overlay you get when putting a PDA in your ID slot; namely, because it's an eyesore, and despite the fact i'm sure someone will manifest to tell me it's actually extremely important for X, i'll pre-emptively say we've been without it for like, two years at this rate, and nothing negative ever came from it's removal, just a ton of positives.

## How This Contributes To The Skyrat Roleplay Experience
the worn icon is an actual eyesore

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog

:cl:
del: removed the PDA worn icon; again
/:cl:

### CODE REVIEW

**Comment by RatFromTheJungle** [comment]
you were too slow bot everyone already saw the crab

**Comment by GoldenAlpharex** [comment]
I tried to do it yesterday and gave up after it didn't work twice.

File: code/modules/mob/living/carbon/human/human_update_icons.dm [hunk 1]
```diff
@@ -141,18 +141,20 @@ There are several things that need to be remembered:
...
 	if(wear_id)
 		var/obj/item/worn_item = wear_id
 		update_hud_id(worn_item)
-		var/icon_file
+		//var/icon_file SKYRAT EDIT REMOVAL - PDA-B-GONE
+//SKYRAT EDIT REMOVAL BEGIN - PDA-B-GONE
+/*
 
 		if(!icon_exists(icon_file, RESOLVE_ICON_STATE(worn_item)))
 			icon_file = 'icons/mob/clothing/id.dmi'
```
Comment (GoldenAlpharex) [inline code/modules/mob/living/carbon/human/human_update_icons.dm]: The only line that actually needs commenting should be this one.

**Review by GoldenAlpharex — DISMISSED** [review]

File: code/modules/mob/living/carbon/human/human_update_icons.dm (hunk 1 above)
Comment (RatFromTheJungle) [reply code/modules/mob/living/carbon/human/human_update_icons.dm]: if everything below it is uncommented (also, I tried the various line across this error on their own individually, and didn't get much luck without something else breaking): 
image

if 155 isn't commented, the id still shows, and if the two above it aren't commented (150, 144), the id turns invisible in the slot

Comment (GoldenAlpharex) [reply code/modules/mob/living/carbon/human/human_update_icons.dm]: There's a mirror coming from upstream that's going to change these lines, I'll merge it probably today, and then it should be as simple as changing one lines, really.

Comment (RatFromTheJungle) [reply code/modules/mob/living/carbon/human/human_update_icons.dm:160]: 57inygvse5n51

Comment (RatFromTheJungle) [reply code/modules/mob/living/carbon/human/human_update_icons.dm]: image

**Comment by tf-4** [comment]
Wait no I misread

**Review by tf-4 — DISMISSED** [review]
Do the thing that golden said

**Comment by RatFromTheJungle** [comment]
> Do the thing that golden said

i tried it see the comment below it, all variations but as-done either
made the ID/pda entirely invisible if you put it into your id slot (couldn't get it back out) 
or 
didn't remove the icon
or
turned you into a giant error when wearing a pda bro there aint' nothing else I can do bro

**Comment by tf-4** [comment]
You tried it since the upstream PR they talked about? I'll be honest I'm not sure what they mean though so it'll need @GoldenAlpharex to weigh in

File: code/modules/mob/living/carbon/human/human_update_icons.dm [hunk 2]
```diff
@@ -148,19 +148,19 @@ There are several things that need to be remembered:
...
+	// var/mutable_appearance/id_overlay = overlays_standing[ID_LAYER] SKYRATEDIT - PDA-B-GONE
 
 	if(wear_id)
 		var/obj/item/worn_item = wear_id
 		update_hud_id(worn_item)
-		var/icon_file = 'icons/mob/clothing/id.dmi'
-
+		// var/icon_file = 'icons/mob/clothing/id.dmi' SKYRATEDIT - PDA-B-GONE
+/* SKYRAT REMOVAL BEGIN - PDA-B-GONE
 		id_overlay = wear_id.build_worn_icon(default_layer = ID_LAYER, default_icon_file = icon_file)
```
Comment (GoldenAlpharex) [inline code/modules/mob/living/carbon/human/human_update_icons.dm:164]: This should be all you need.
```suggestion
		/* SKYRAT EDIT REMOVAL - PDA-B-GONE
		id_overlay = wear_id.build_worn_icon(default_layer = ID_LAYER, default_icon_file = icon_file)
		*/ // SKYRAT EDIT END
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/human_update_icons.dm (hunk 2 above)
Comment (RatFromTheJungle) [reply code/modules/mob/living/carbon/human/human_update_icons.dm:164]: that did work, I probably just did it wrong the last time i tested

Comment (RatFromTheJungle) [reply code/modules/mob/living/carbon/human/human_update_icons.dm:164]: that did work, I probably just did it wrong the last time i tested, however, without the line above taken out, it still fails windowsbuild

File: code/modules/mob/living/carbon/human/human_update_icons.dm [hunk 3]
```diff
@@ -159,10 +159,10 @@ There are several things that need to be remembered:
 	if(wear_id)
 		var/obj/item/worn_item = wear_id
 		update_hud_id(worn_item)
-		var/icon_file = 'icons/mob/clothing/id.dmi'
-
+		//var/icon_file = 'icons/mob/clothing/id.dmi' SKYRAT EDIT REMOVAL - PDA-B-GONE
+		/* SKYRAT EDIT START - PDA-B-GONE
```
Comment (GoldenAlpharex) [inline code/modules/mob/living/carbon/human/human_update_icons.dm]: Smh jungle, going with the lazy option.
```suggestion
		/* SKYRAT EDIT START - PDA-B-GONE
		var/icon_file = 'icons/mob/clothing/id.dmi'
```

File: code/modules/mob/living/carbon/human/human_update_icons.dm [hunk 4]
```diff
@@ -172,6 +172,7 @@ There are several things that need to be remembered:
 
 	apply_overlay(ID_LAYER)
 
+
```
Comment (GoldenAlpharex) [inline code/modules/mob/living/carbon/human/human_update_icons.dm]:
```suggestion
```

**Review by GoldenAlpharex — APPROVED** [review]
Should be good to go now!

**Pull request merged**